index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en-GB">
+<head>
+    <script type="text/javascript" src="" async></script>
+    <link href="https://fonts.googleapis.com/css?family=Press+Start+2P&display=swap" rel="stylesheet">
+    <link rel="stylesheet" type="text/css" href="normalize.css">
+    <link rel="stylesheet" type="text/css" href="styles.css">
+    <title>Match the Geckos!</title>
+</head>
+
+<body>
+<div class="container">
+    <div class="menu">
+        <button class="instructionsButton">Instructions</button>
+        <button class="newGame">New Game</button>
+        <div class="score"></div>
+        <div class="timer"></div>
```

Comment:
I get why these divs are here, but technically speaking empty divs aren't great. I'd remove these and just leave space to add them in later if/when we accept more stories.